www-servers/sniproxy/sniproxy-0.6.0.ebuild

```diff
@@ -37,6 +39,8 @@ DEPEND="
 	test? ( net-misc/curl )
 "
```

Comment:
FYI, these dependency specs looks wrong... libev and libpcre in BDEPEND? :raised_eyebrow:

Reply:
Good catch, I don't know how it ended up here, I guess ${RDEPEND} meant to be included in DEPEND instead of BDEPEND

Reply:
Fixed it